Angular/login/src/app/services/ipo.service.ts

```diff
@@ -0,0 +1,51 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ibmpariwesh/socgen-stockmarket-project/pullRequest?prid=8405403)
```suggestion
import { HttpClient, HttpHeaders } from "@angular/common/http";
```